src/server/components/lists/controllers/ingest/helpers.ts

```diff
@@ -18,7 +18,17 @@ export function arrayHasChanges(beforeArray: any[] = [], afterArray: any[] = [])
     return false;
```

Comment:
As pointed out above, I think this is incorrect as it should detect if an array was empty before something was populated in it but this line would automatically return false if beforeArray is empty.